docs/artists/decaryi.md

```diff
@@ -1,22 +1,22 @@
 # Decaryi
 
-[[img|artists/images/decaryi.jpg|400]]
+![](images/decaryi.jpg ':size=400')
 
 Decaryi is a musical project, created by Anton Schmidt, based in Augsburg. He started to make music as a teen by playing guitar in some metal bands but as his music taste expands, his style of creating
 musig changed. Music from King Crimson, Can and Nicolas Jaar inspired him to make more progressive
 music. Also improvisation started to be a big part of his music, as well as experimental ways of creating sound.
 
 His music is a mix between electronic music, like trip hop, ambient and dub with rock music, like kraut -, psychedelic - or stoner rock. He works with several techniques to create his music. But most of the time he uses guitars, modular synths or field recorded samples.
 
-!! Links
+# Links
 
 * [[https://www.youtube.com/channel/UCLGnmbBp-H-i05V_ZtwroNw | YouTube]]
 * [[https://decaryi.bandcamp.com/SoundCloud: | Bandcamp]]
 * [[https://www.instagram.com/_decaryi_/ | Instagram]]
 * [[https://open.spotify.com/artist/5K2mtKL5OmEB5ajctNLpov?si=kXojkM1DTQKsUwrYSviMew | Spotify]]
 * [[https://soundcloud.com/user-484887923 | Soundcloud]]
 
-'''YouTube''' 
+<b>YouTube</b> 
```

Comment:
So Markdown does support markup shortcuts for bold and italics and I would prefer this use over the more verbose Html. According to this document (https://www.markdownguide.org/basic-syntax/) the following markup should be used instead:
`I just love **bold text**.`
and
`Italicized text is the *cat's meow*.`

Reply:
Makes total sense.  The Ai should have thought of that :sweat_smile: I just applied the funtions it wrote :P 
I will fix